Examples/Tests/embedded_circle/inputs_2d

```diff
@@ -37,8 +37,8 @@ electrons.density = 10.2e16
 electrons.profile = constant
 electrons.momentum_distribution_type = maxwell_boltzmann
 electrons.theta = (kb*30000/(m_e*clight^2))
-electrons.save_particles_at_zlo = 1
-electrons.save_particles_at_zhi = 1
+electrons.save_particles_at_xlo = 1
+electrons.save_particles_at_xhi = 1
```

Comment:
Thanks for the comments!
I don't know why this test case was setting `electrons.save_particles_at_zlo` and `hi` since it has periodic boundary conditions in z, so no particles are ever lost in z. Particles are lost in x however, so I changed it to x. I can update the analysis.py file in this test to check that the output of the scraped particles is correct.